admin/tools/docker/replication/container/tools/README

```diff
@@ -6,54 +6,27 @@
 # Go into a development folder where you normally build Qserv (by running
 # the scons tool)
 
-cd /home/user/development/qserv
+cd /home/<user>/development/qserv
 
-# Use the previously built development container to collect binaries along
-# with their library dependencies at the temporary container build area
-
-docker run \
-    --rm \
-    -u "$(id -u):$(id -g)" \
-    -e "SOURCE=${PWD}" \
-    -v /etc/passwd:/etc/passwd:ro \
-    -v /etc/group:/etc/group:ro \
-    -v $PWD:$PWD \
-    qserv/replica:dev \
-    bash -c '$SOURCE/admin/tools/docker/replication/container/tools/collect.sh $SOURCE'
-
-
-# Build the runnable container from the previously prepared area
-
-docker build \
-   -t qserv/replica:tools \
-   -f $PWD/admin/tools/docker/replication/container/tools/Dockerfile.ncsa \
-   tmp/replication/container/build
-
-# (optional) run a few tests to inspect if container content is well formed
-# and applications' dependencies are satisfied.
-
-docker run --rm qserv/replica:tools ls -al /qserv/lib
-docker run --rm qserv/replica:tools ls -al /qserv/bin
-docker run --rm qserv/replica:tools ldd /qserv/bin/qserv-replica-job-chunks
-docker run --rm qserv/replica:tools /qserv/bin/qserv-replica-job-chunks
-
-
-# And, finally - publish the container (provided a proper login into DockerHub was
-# made earlier)
-
-docker push qserv/replica:tools
-
-####################################################################
-# An alternative method is run the following script wich will do all
-# explained above and build a container tagged as 'qserv/replica:tools'.
-
-admin/tools/docker/replication/container/tools/build.sh $PWD ncsa
-
-# Or, if the default container tag 'qserv/replica:tools' is not desired, using
-# the Git hash to tag the container. Note that this container will be built using
-# a different specification file 'Dockerfile.gke'. Look inside 'Dockerfile.gke'
-# to see how it's different from 'Dockerfile.ncsa'.
+# Run the following script that will collect dependencies (libraries into
+# a temporary folder) and build a container tagged with the specified tag.
+# Examples below making a tag based on the git hash.
+#
+# Note the optional flag that would tell the script that a definition of
+# the current user is found in the system's files:
+#
+#   /etc/passwd
+#   /etc/group
+#
+# Without the flag the script will expected similar files be found at
+# the user's home folder:
+#
+#   ~/passwd
+#   ~/group
+#
+# User definition is needed to run the Docker container collecting dependencies
+# under the current user.
 
 TAG="qserv/replica:tools-$(git describe --dirty --always)" 
-admin/tools/docker/replication/container/tools/build.sh $PWD gke "$TAG"
-
+admin/tools/docker/replication/container/tools/build.sh $PWD "$TAG"
+admin/tools/docker/replication/container/tools/build.sh $PWD "$TAG" --local-user
```

Comment:
One of them, depending on if there is a user entry in the local `/etc/passwd` file.
I should also note that this mechanism for building the binary containers is only used at NCSA. I was deployed 2 years ago to avoid dealing with monstrous 5 GB containers. It's supposed to be replaced with Nate's `lite` version.